packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/ImageAnalysisProxyApi.java

```diff
@@ -18,11 +23,13 @@
 class ImageAnalysisProxyApi extends PigeonApiImageAnalysis {
   static final long CLEAR_FINALIZED_WEAK_REFERENCES_INTERVAL_FOR_IMAGE_ANALYSIS = 1000;
 
+  @OptIn(markerClass = ExperimentalCamera2Interop.class)
```

Comment:
This plugin update [lint-baseline.xml] (https://github.com/flutter/packages/blob/main/packages/camera/camera_android_camerax/android/lint-baseline.xml) to handle this specific warning.

This should be updateable by following https://developer.android.com/studio/write/lint#snapshot in the example app. But this can be updated after getting both LGTMs.